iwyu_tool.py

```diff
@@ -496,8 +496,11 @@ def custom_help():
     parser.add_argument('-o', '--output-format', type=str,
                         choices=FORMATTERS.keys(), default=DEFAULT_FORMAT,
                         help='Output format (default: %s)' % DEFAULT_FORMAT)
-    parser.add_argument('-j', '--jobs', type=int, default=1,
-                        help='Number of concurrent subprocesses')
+    parser.add_argument('-j', '--jobs', type=int, default=0,
```

Comment:
It's a bit tricky, but I'd like to see the absence of `-j` evaluate to jobs == 1, and `-j` without a value evaluate to jobs == 0 (which is further refined to `os.cpu_count()`).

Is `nargs='?'` required to make the integer value optional? I'm not sure what effect `const=0` has here.

I can't think of an argparse flag configuration to allow `() => 1`, `-j => 0`, `-j4 => 4`, so I wonder if it's easier to just remove all the argparse settings and check `args.jobs` using `hasattr`/`getattr`

Reply:
`const=0` determines the value if the argument is used as a flag (`-j` case, as opposed to `-jX`). To get what you want, I just need to change to `default=1, const=0`.

Reply:
Ah, nice. I learn something new about argparse every time I use it :upside_down_face: